src/analysis/Environments/UnsupportedWorkspaceCell.test.tsx

```diff
@@ -0,0 +1,15 @@
+import { screen } from '@testing-library/react';
+import React from 'react';
+import { renderWithAppContexts as render } from 'src/testing/test-utils';
+
+import { UnsupportedWorkspaceCell } from './UnsupportedWorkspaceCell';
+
+test('renders UnsupportedWorkspaceCell with status and message', () => {
```

Comment:
Nit: for convention, let's wrap tests for a component in a describe block named after the component.

And super nit: most tests use `it` instead of `test`.

```suggestion
describe('UnsupportedWorkspaceCell', () => {
  it('renders UnsupportedWorkspaceCell with status and message', () => {
```